sdl2promela/translator.py

```diff
@@ -742,7 +742,7 @@ def __generate_input_function(
     # Generate a call to execute transition inline.
     # The execution id depends on current state, which includes parallel states.
     # Start from top_level state: i.e. the variable 'state' in process context.
-    for statename in top_level_states:
+    for statename in sorted(top_level_states):
```

Comment:
Why the sorting is added?

Reply:
For tests. I've added a test to check if translator can generate a code which initializes variables declared in nested state. The presence of nested state causes fails of the test, because the order of generated alternatives is different. Sorting solves the issue.